### DISCUSSION TITLE
Is there a way to make cmd-click in VSCode resolve the actual component?

### DISCUSSION THREAD

**Discussion by TheDutchCoder** [nuxt/nuxt#16890] [Questions]
Currently cmd-clicking an auto imported component will open `components.d.ts` which itself only references the vue shim.
It would be nice if we could open the actual component directly.

**Comment by benoitdemaegdt** [comment]
Please check this vscode extension : https://github.com/antfu/vscode-goto-alias
It is not a perfect solution because there is a "flash" happening when redirecting, but much better than nothing 😉
